src/dag_isomorphism.rs

```diff
@@ -190,13 +194,44 @@ pub fn is_isomorphic(dag0: &PyDiGraph, dag1: &PyDiGraph) -> PyResult<bool> {
     Ok(res.unwrap_or(false))
 }
 
+fn clone_graph(py: Python, dag: &PyDiGraph) -> PyDiGraph {
```

Comment:
Maybe `reindex_graph`? When I got 247 and 253 it read like `if a node was removed, assign to copy(g), else assign to g`

Reply:
yep, I agree that's a better name I'll fix it now

Reply:
Fixed in: https://github.com/Qiskit/retworkx/pull/115/commits/a1da585a06acd81b20f346ab666263e6c9aa2075